### PR TITLE
Add user note edit view for Hathor

### PR DESCRIPTION
**Pull request by n9iels**
This PR solves issue #7836. It will add a template override to the Hathor template for the com_users/note/edit.php
## How to test

1) Set Hathor as default admin template
2) Go to Users -> User Notes and press the new button
3) Before applying the patch you will see this:
687474703a2f2f6973737565732e6a6f6f6d6c612e6f72672f75706c6f6164732f312f64383939326132313762383265303263333535633332383636363065373362612e706e67

After applying the patch:
knipsel

4) Also confirm the layout is the same for editing an user note

### CODE REVIEW

File: administrator/language/en-GB/en-GB.com_users.ini [hunk 1]
```diff
@@ -196,6 +196,7 @@ COM_USERS_N_USERS_UNBLOCKED="%s Users enabled."
 COM_USERS_N_USERS_UNBLOCKED_0="No User enabled."
 COM_USERS_N_USERS_UNBLOCKED_1="User enabled."
 COM_USERS_NEW_NOTE="New Note"
+COM_USERS_EDIT_NOTE="Edit Note"
```
Comment (zero-24) [inline administrator/language/en-GB/en-GB.com_users.ini]: Can you alpha sort that string?

**Comment by n9iels** [comment]
@zero-24 alpha sort the language file. No idea why that line 317 came up

**Comment by KingLouis1** [comment]
I have tested this item :white_check_mark: successfully on cb4be35d6fbed08170b2fadbd19a3956a7acd61e<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/8066">issues.joomla.org/joomla-cms/8066</a>.</sub>

**Comment by KingLouis1** [comment]
I have tested this item :white_check_mark: successfully on cb4be35d6fbed08170b2fadbd19a3956a7acd61e<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/8066">issues.joomla.org/joomla-cms/8066</a>.</sub>

**Comment by webberry** [comment]
I have tested this item :white_check_mark: successfully on cb4be35d6fbed08170b2fadbd19a3956a7acd61e<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/8066">issues.joomla.org/joomla-cms/8066</a>.</sub>

**Comment by SniperSister** [comment]
2 successful tests, setting RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/8066">issues.joomla.org/joomla-cms/8066</a>.</sub>

**Comment by Kubik-Rubik** [comment]
Thank you @n9iels! Merged.

**Pull request merged**